codelabs/custom_generator/custom_generator.md

```diff
@@ -108,14 +114,27 @@ Blockly.common.defineBlocksWithJsonArray([{
 }]);
 ```
 
+This code creates the block definitions, but it doesn't register them with Blockly. We'll do that in `src/index.js`.
```

Comment:
Maybe explain what "register with Blockly" means. It's not clear to a newcomer.

Reply:
Also, say where in `index.js` this code goes.

Reply:
I tried to explain more, not sure if I made it clearer. I think it's explained below when I show the code that does register the definitions.